### ISSUE TITLE
Hard coded paths throughout project

### ISSUE THREAD

**Issue by nelsonej** [AliciaBentley-NOAA/gefs_plumes#1]
I don't know if it's intended (or even possible) to run the GEFS plumes webpage outside of NOAA servers, but was interested in running the software for my personal use.  While I can easily "find/replace" some or most of these paths, I'm a little surprised at the extent of the hard coded paths throughout the project.

Is there a plan to update the software or is the initial dump all we get?

Thanks!

**Comment by AliciaBentley-NOAA** [comment]
You are correct- there will not be additional updates made to this repository going forward due to resource constraints at NOAA/NWS/EMC. The [GEFS Plumes webpage](https://www.emc.ncep.noaa.gov/users/meg/gefs_plumes/index.html) code was uploaded to GitHub in its current state when the decision to turn off the webpage on 10/1/21 was made. While not ideal, this repository was thought to be better than turning off the webpage and providing the community with nothing. I know a few people who are making this webpage work outside of NOAA servers, and I'd be happy to include those links in this thread after they're webpages are up and running! Thank you for your interest.

**Comment by nelsonej** [comment]
I understand and thank you for making the repository available. If/when I have time to make modifications I'll fork and make available (no promises).
